packages/console/src/components/ImageInputs/index.tsx

```diff
@@ -40,6 +45,8 @@ export type ImageField<FormContext extends FieldValues> = {
 type Props<FormContext extends FieldValues> = {
   /** The condensed title when user assets service is available. */
   readonly uploadTitle: React.ComponentProps<typeof FormField>['title'];
+  /** The tooltip to show for all the fields. */
```

Comment:
nit:
`!isUserAssetServiceReady`: for all input fileds
`isUserAssetServiceReady`: for the form field section

Reply:
they are included in "all the fields", right?

Reply:
LOG-9596